code/game/gamemodes/clock_cult/clock_cult.dm

```diff
@@ -178,13 +178,16 @@ Credit where due:
 	return 1
 
 /datum/game_mode/clockwork_cult/post_setup()
+	var/list/spread_out_spawns = GLOB.servant_spawns
 	for(var/S in servants_to_serve)
+		if(!spread_out_spawns.len) //cycle through the list again if we've used all the inital spawnpoints.
+			spread_out_spawns = GLOB.servant_spawns
```

Comment:
This isn't copying, the `.Remove` later will remove from `GLOB.servant_spawns` too